news/1805.feature

```diff
@@ -0,0 +1 @@
+Add `--experimental-rerun` command-line option
```

Comment:
... to reproduce pickled single runs

Reply:
fixed in [b6ae6ed](https://github.com/facebookresearch/hydra/pull/2098/commits/b6ae6eda6d2a7d8a68a6ffda221acfb40cc96a95)